src/odin/ingestion/afc/afc_archive.py

```diff
@@ -91,30 +91,36 @@ class ArchiveAFCAPI(OdinJob):
     def __init__(self, table: str) -> None:
         """Create Job instance."""
         self.table = table
-        self.start_kwargs = {"table": self.table}
-        self.export_folder = os.path.join(DATA_SPRINGBOARD, AFC_DATA, self.table)
+        self.start_kwargs = {"table": table}
+        self.export_folder = os.path.join(DATA_SPRINGBOARD, AFC_DATA, table)
         self.headers = {
             "client_id": os.getenv("AFC_API_CLIENT_ID", ""),
             "client_secret": os.getenv("AFC_API_CLIENT_SECRET", ""),
         }
 
     def make_request(
-        self, url: str, fields: Optional[Dict[str, str]] = None
+        self,
+        url: str,
+        method: str = "GET",
+        fields: dict[str, str] | None = None,
+        preload_content: bool = True,
     ) -> urllib3.BaseHTTPResponse:
         """
         Make AFC API Request.
 
         Will raise if 200 status is not returned.
 
-        :param url: full url to be used in GET Request.
-        :param fields: (Optional) params/fields to be included in GET request.
+        :param url: full url to be used in Request.
+        :param method: (Optional) HTTP request method (such as GET, POST, PUT, etc.)
+        :param fields: (Optional) Data to encode and send in request body.
+        :param preload_content: (Optional) If True (default) response body preloaded into memory.
```

Comment:
I think we want to use this `preload_content` parameter, but i'll monitor resource usage and see if this is a positive change.